execution/executor/src/components/chunk_output.rs

```diff
@@ -33,8 +33,14 @@ impl ChunkOutput {
         transactions: Vec<Transaction>,
         state_view: CachedStateView,
     ) -> Result<Self> {
+        let start = Instant::now();
         let transaction_outputs = V::execute_block(transactions.clone(), &state_view)?;
 
+        debug!(
```

Comment:
remove?

Reply:
I was planning to leave it in, but I can remove it